### ISSUE TITLE
Integration test failure: Unknown namespace 'ff_system'

### ISSUE THREAD

**Issue by awrichar** [hyperledger/firefly#1245]
https://github.com/hyperledger/firefly/actions/runs/4559216926/jobs/8042927545

Migration tests for legacy namespaces that rely on "ff_system" are failing. Recent namespace manager work may have broken the functionality for "V1" namespaces.

**Comment by awrichar** [comment]
Seems that when we query for network version, we look for a specific error to indicate that the query failed:
https://github.com/hyperledger/firefly/blob/df2581ef959e6c7cce7ef299bc959411d14e201e/internal/blockchain/ethereum/ethereum.go#L961

This is the error in ethconnect:
https://github.com/hyperledger/firefly-ethconnect/blob/da2de300ee2adeefb95fc412df8ce8c080645860/internal/errors/errors.go#L363

This bug is apparently related to the switch from ethconnect to evmconnect as the default connector.

ethconnect returned a specific and descriptive error when attempting to query a method that does not exist: `FFEC100148: Call failed: execution reverted`.
On the other hand, evmconnect just returns a more generic error message of `execution reverted`.

**Comment by awrichar** [comment]
I've confirmed that the combination of #1246 + #1249 allows this test to pass.

I also confirmed locally that this was broken in FireFly v1.2.0 - which implies we did not run the integration suite on that release. It could warrant a v1.2.1 with these fixes, but that would need to be pulled from a release branch of FireFly and not `main`, as there are currently some larger changes being worked in mainline. The only complication would be whether the latest version of evmconnect could be used, since part of the fix is a required change in that repo (and it's seen a number of commits+releases since the v1.2.0 FireFly release).
